### PR TITLE
Move To Master

### PR DESCRIPTION
**Pull request by liferay-commerce**

### CODE REVIEW

**Comment by liferay-commerce** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by pyoo47** [comment]
ci:test:sf

**Comment by liferay-commerce** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:stable

**Comment by jpince** [comment]
@marco-leo @dsitu @gianmarcobrunialti compile error

**Comment by dsitu** [comment]
> @marco-leo @dsitu @gianmarcobrunialti compile error

I have a fix that ill send to Marco.

**Comment by jpince** [comment]
@dsitu @marco-leo it looks like the branch is missing the `modules/.releng` directories. I thought I saw them in a previous iteration. Maybe the rebase script doesn't include them?
https://github.com/liferay/liferay-portal-ee/tree/7.1.x/modules/.releng/apps/commerce
https://github.com/liferay/liferay-portal-ee/tree/7.1.x/modules/.releng/dxp/apps/commerce
`app.portal.build` should be set to `7305` for `modules/.releng/apps/commerce/app.properties` and `7310` for `modules/.releng/dxp/apps/commerce/app.properties`

@petershin can you please validate the branch for correct `.bnd` version numbers? Anything else obvious we should be checking for in copying commerce from 7.1.x to master?

**Comment by petershin** [comment]
@jpince - The major version needs to bumped by 1.  For example: `commerce-account-admin-web` should be `2.0.0`.

Apps in `master` are usually 2 major versions ahead of `7.1.x`.  If the commerce apps will be added to 7.2.x later on, then the major version should be bumped by 2.

```
7.1.x -> 1.0.0
7.2.x -> 2.0.0
7.3.x / master-> 3.0.0
```

**Comment by petershin** [comment]
@jpince - I also noticed `apply plugin: "com.liferay.lang.merger"` in a few `build.gradle` files

The `com.liferay.lang.merger` Gradle plugin was removed: https://github.com/brianchandotcom/liferay-portal/pull/92863

File: modules/.releng/apps/commerce/app.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 app.marketplace.id=123396799
 app.marketplace.title=Liferay CE Commerce
 app.marketplace.version=2.0.7
-app.portal.build=7102
\ No newline at end of file
+app.portal.build=7310
```
Comment (jpince) [inline modules/.releng/apps/commerce/app.properties:4]: @dsitu this is for the CE app, so it needs to be 7305 as mentioned in my [previous comment](https://github.com/liferay-commerce/liferay-portal/pull/2#issuecomment-675668813)

**Comment by marco-leo** [comment]
ci:test:sf

**Comment by jpince** [comment]
^approval was for app.properties changes in 4f76ab5895a6c6aa172ce97ef99730e091da3c51

**Comment by marco-leo** [comment]
ci:test:stable

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:stable

**Comment by jpince** [comment]
ci:test:stable

**Comment by jpince** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:stop

**Comment by marco-leo** [comment]
sent to https://github.com/liferay-commerce/liferay-portal/pull/35

**Pull request closed without merging**